unity/Assets/Scripts/AgentManager.cs

```diff
@@ -1387,7 +1387,7 @@ public class ObjectMetadata {
                          //
                          // temperature placeholder values, might get more specific later with degrees but for now just track these three states
     public enum Temperature { RoomTemp, Hot, Cold };
-    public string ObjectTemperature;// return current abstracted temperature of object as a string (RoomTemp, Hot, Cold)
+    public string temperature;// return current abstracted temperature of object as a string (RoomTemp, Hot, Cold)
```

Comment:
Could we just have a single temperature Enum? This is ServerAction so it doesn't matter a ton.  Alternatively, you could use this as an opportunity to change all the actions that depend on `Temperature` and `temperature` to the ActionDispatcher style.

Reply:
This was in `ObjectMetadata`, not `ServerAction`, but the same problem still applied, where one shouldn't have both `temperature` and `Temperature`. Although, unexpectedly, `Temperature` was never being returned on the python side (presumably because it was an enum).

In https://github.com/allenai/ai2thor/pull/832/commits/ee94dffa5f2e0762d9ee7f4959e920642d07534c, I moved the two existing enums out of `ObjectMetadata` for consistency, so they don't unexpectedly appear in the metadata if enums are serialized at some point.